art/attacks/evasion/fast_gradient.py

```diff
@@ -84,7 +84,7 @@ def __init__(
         Create a :class:`.FastGradientMethod` instance.
 
         :param estimator: A trained classifier.
-        :param norm: The norm of the adversarial perturbation. Possible values: "inf", np.inf, 1 or 2.
+        :param norm: The norm of the adversarial perturbation. Possible values: "inf", `np.inf` or a real `p >= 1`.
```

Comment:
Please run `black` formatter on the art folder with `black --line-length 120 art/` with version `black==21.12b0`

Reply:
Done. I accidentaly ran black with latest version first, but reran it with `21.12b0` afterwards.